From 4c66aee456ae81c584647bb4f55a9a1bcd5adf26 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Sat, 23 Feb 2019 21:26:15 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Remove WiFi host-wake GPIO

This device does not have any GPIO for host-wake,
the driver will now use SDIO interrupts.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index aa6cb3a78b2d..d7dfa895cf56 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -1190,8 +1190,6 @@
 			reg = <1>;
 			comaptible = "brcm,bcm4329-fmac";
 			interrupt-parent = <&gpio>;
-			interrupts = <TEGRA_GPIO(S, 2) IRQ_TYPE_EDGE_BOTH>;
-			interrupt-names = "host-wake";
 		};
 
 	};
-- 
2.22.0

